lib/unleash/bootstrap/handler.rb

```diff
@@ -0,0 +1,22 @@
+require 'unleash/bootstrap/provider/from_url'
+require 'unleash/bootstrap/provider/from_file'
+
+module Unleash
+  module Bootstrap
+    class Handler
+      attr_accessor :configuration
+
+      def initialize(configuration)
+        self.configuration = configuration
+      end
+
+      # @return [String] JSON string representing data returned from an Unleash server
+      def retrieve_toggles
+        return Provider::FromFile.read(configuration.file_path) unless self.configuration.file_path.nil?
+        return Provider::FromUrl.read(configuration.url, configuration.url_headers) unless self.configuration.url.nil?
+        return configuration.data unless self.configuration.data.nil?
+        return configuration.closure.call if self.configuration.closure.is_a?(Proc)
```

Comment:
We should also document the prio order in the README!


```suggestion
        return configuration.data unless self.configuration.data.nil?
        return configuration.closure.call if self.configuration.closure.is_a?(Proc)
        return Provider::FromFile.read(configuration.file_path) unless self.configuration.file_path.nil?
        return Provider::FromUrl.read(configuration.url, configuration.url_headers) unless self.configuration.url.nil?
```

Reply:
Good suggestion! I've added a warning to the README that you should only select one, along with a description of how they're resolved if multiple are specified